test/orm_test.rb

```diff
@@ -80,12 +80,12 @@ def test_create
     end
 
     def test_create_with_server_returning_validation_errors
-      endpoint = stub_request(:put, 'http://sushi.com/recipes/1').to_return_json(id: 'write_error:400', errors: { title: [{ error: 'too_short', count: 4 }], groups: [{ error: 'blank' }] })
+      endpoint = stub_request(:put, 'http://sushi.com/recipes/1').to_return_json(id: 'write_error:400',
+        errors: { title: [{ error: :too_short, count: 4 }, { error: 'plain text', use_i18n: false }], groups: [{ error: 'blank' }] })
```

Comment:
not really happy about introducing another "non-standard" option, `:use_i18n` here.. but since these are transferred with http request, even though you pass symbol like `:too_short` in the example, when handling it in the `add_errors_to_model` method it'll become an instance of String so can't rely on `error.is_a?(Symbol)` to check it, any better idea?